recipes/poco/all/patches/1.8.1.patch

```diff
@@ -0,0 +1,59 @@
+--- cmake/PocoMacros.cmake
++++ cmake/PocoMacros.cmake
+@@ -265,7 +265,7 @@
+ 
+ if (MSVC)
+ # install the targets pdb
+-  POCO_INSTALL_PDB(${target_name})
++  # POCO_INSTALL_PDB(${target_name})
+ endif()
+   
+ endmacro()
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -204,7 +204,7 @@
+ 
+ find_package(OpenSSL)
+ if(OPENSSL_FOUND)
+-    include_directories("${OPENSSL_INCLUDE_DIR}")
++    include_directories("${OpenSSL_INCLUDE_DIR}")
+     if(EXISTS ${PROJECT_SOURCE_DIR}/NetSSL_OpenSSL AND ENABLE_NETSSL)
+     add_subdirectory(NetSSL_OpenSSL)
+     list(APPEND Poco_COMPONENTS "NetSSL_OpenSSL")
+--- Crypto/CMakeLists.txt
++++ Crypto/CMakeLists.txt
+@@ -20,7 +20,7 @@
+     DEFINE_SYMBOL Crypto_EXPORTS
+     )
+ 
+-target_link_libraries( "${LIBNAME}" Foundation ${OPENSSL_LIBRARIES} )
++target_link_libraries( "${LIBNAME}" Foundation OpenSSL::OpenSSL )
+ target_include_directories( "${LIBNAME}"
+     PUBLIC
+         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+--- Data/SQLite/CMakeLists.txt
++++ Data/SQLite/CMakeLists.txt
+@@ -10,9 +10,9 @@
+ POCO_HEADERS_AUTO( SQLITE_SRCS ${HDRS_G})
+ 
+ if (POCO_UNBUNDLED)
+-    find_package(SQLite3)
+-    set(DATASQLITELIBS ${SQLITE3_LIBRARIES})
+-    include_directories("${SQLITE3_INCLUDE_DIRS}")
++    find_package(SQLite3 REQUIRED)
++    set(DATASQLITELIBS ${SQLite3_LIBRARIES})
++    include_directories(${SQLite3_INCLUDE_DIRSS})
```

Comment:
If we are linking with the targets, needed?

Reply:
Not really, but this is closer to the original code.